Scripts/NodeDataCache.cs

```diff
@@ -150,5 +183,10 @@ private class PortDataCache : Dictionary<System.Type, List<NodePort>>, ISerializ
                     this.Add(keys[i], values[i]);
             }
         }
+
+        [MenuItem("TOOLS/LOL")]
+        public static void LOL() {
+            portDataCache = null;
+        }
```

Comment:
Leads to belly cramps when invoked too many times. Perhaps we should lay it off for now.

Reply:
This is an oversight; I was using this to poke around the cache rebuilding method more easily for informative purposes, but I shouldn't have forgotten to delete this method entirely...

Reply:
Now removed.